lib/ansible/cli/pull.py

```diff
@@ -42,6 +42,8 @@ class PullCLI(CLI):
         This is useful both for extreme scale-out as well as periodic remediation.
         Usage of the 'fetch' module to retrieve logs from ansible-pull runs would be an
         excellent way to gather and analyze remote logs from ansible-pull.
+        
+        Note that the playbook is automatically limited to localhost by a hidden --limit flag. You can add your own limit flag to override.
```

Comment:
localhost or the hostname of the machine on which ansible-pull executes, as it is designed to run playbooks that affect the local machine